### ISSUE TITLE
Automatic replies using DialogFlow

### ISSUE THREAD

**Issue by pedrofracassi** [SwitchbladeBot/switchblade#570]
Much like [jagrosh's phoenix bot](https://github.com/jagrosh/Phoenix), we could let users connect the bot to DialogFlow and make it answer automatically on a support channel, for example.

**Comment by perronosaurio** [comment]
i like this

**Comment by pedrofracassi** [comment]
![image](https://user-images.githubusercontent.com/25179120/50792764-3fc49200-12ad-11e9-8ba8-189d59cfcd0c.png)

**Comment by Doges** [comment]
> Much like [jagrosh's phoenix bot](https://github.com/jagrosh/Phoenix), we could let users connect the bot to DialogFlow and make it answer automatically on a support channel, for example.

how we can get this with crowdin?

**Comment by pedrofracassi** [comment]
It wouldn't be a feature exclusive to bladey-related questions. Users would be able to link their dialogflow accounts(?) and add their own automated answers to their servers/channels. We could still use it on our server for bladey-related stuff, tho.

**Comment by Doges** [comment]
> It wouldn't be a feature exclusive to bladey-related questions. Users would be able to link their dialogflow accounts(?) and add their own automated answers to their servers/channels. We could still use it on our server for bladey-related stuff, tho.

sooo, how we can deal with the nsfw phrases, like, if someone puts an nsfw url in a dialog and bladey send it in a non-nsfw chat and someone report it

**Comment by pedrofracassi** [comment]
Dialogs would be set up per channel/server, so it's up to the server owners not to put NSFW stuff in them, just like join/leave messages, or anything customizable, really.

**Comment by Doges** [comment]
> Dialogs would be set up per channel/server, so it's up to the server owners not to put NSFW stuff in them, just like join/leave messages, or anything customizable, really.

but if a owner puts a nsfw stuff and someone didn't liked it and report?

**Comment by pedrofracassi** [comment]
That's their problem, not ours. Many bots have customizable messages (Loritta, Mee6, Dyno...), and they're totally fine.

**Comment by Doges** [comment]
> That's their problem, not ours. Many bots have customizable messages (Loritta, Mee6, Dyno...), and they're totally fine.

tbh, just remove the urls

**Comment by pedrofracassi** [comment]
No, URLs would be super useful in this scenario, like linking to a server's facebook page when someone asks for it

`What's the name of the facebook page?`
**Bladey:** `http://facebook.com/...`

**Comment by pedrofracassi** [comment]
ta-da

![image](https://user-images.githubusercontent.com/25179120/50908608-348a7700-1411-11e9-8795-cb46fb13a8ef.png)

**Comment by Doges** [comment]
ta ok

**Comment by dpaiv0** [comment]
get fucking owned
